composeml/tests/test_label_maker.py

```diff
@@ -1,75 +1,70 @@
-# flake8:noqa
 import pandas as pd
 import pytest
 
 from ..label_maker import LabelMaker
 
 
-@pytest.fixture
-def transactions():
-    records = [
-        {
-            'transaction_time': '2014-01-01 00:44:00',
-            'amount': 21.35,
-            'customer_id': 1
-        },
-        {
-            'transaction_time': '2014-01-01 00:45:00',
-            'amount': 108.11,
-            'customer_id': 1
-        },
-        {
-            'transaction_time': '2014-01-01 00:46:00',
-            'amount': 112.53,
-            'customer_id': 1
-        },
-        {
-            'transaction_time': '2014-01-01 00:47:00',
-            'amount': 6.29,
-            'customer_id': 1
-        },
-        {
-            'transaction_time': '2014-01-01 00:48:00',
-            'amount': 47.95,
-            'customer_id': 1
-        },
-        {
-            'transaction_time': '2014-01-01 00:00:00',
-            'amount': 127.64,
-            'customer_id': 2
-        },
-        {
-            'transaction_time': '2014-01-01 00:01:00',
-            'amount': 109.48,
-            'customer_id': 2
-        },
-        {
-            'transaction_time': '2014-01-01 00:02:00',
-            'amount': 95.06,
-            'customer_id': 2
-        },
-        {
-            'transaction_time': '2014-01-01 00:03:00',
-            'amount': 78.92,
-            'customer_id': 2
-        },
-        {
-            'transaction_time': '2014-01-01 00:04:00',
-            'amount': 31.54,
-            'customer_id': 2
-        },
-    ]
-
-    df = pd.DataFrame.from_records(records)
-    return df
-
-
-def test_search_by_time(transactions, labels):
-    def my_labeling_function(df_slice):
-        label = df_slice['amount'].sum()
-        label = label or pd.np.nan
-        return label
+def my_labeling_function(df_slice):
+    label = df_slice['amount'].sum()
+    label = label or pd.np.nan
+    return label
+
+
+def test_search_offset_mix_0(transactions, labels):
+    lm = LabelMaker(
+        target_entity='customer_id',
+        time_index='transaction_time',
+        labeling_function=my_labeling_function,
+        window_size='2min',
+    )
+
+    given_labels = lm.search(
+        transactions,
+        minimum_data='1min',
+        num_examples_per_instance=4,
+        gap='3min',
+    )
+
+    pd.testing.assert_frame_equal(given_labels, labels)
+
+
+def test_search_offset_mix_1(transactions, labels):
```

Comment:
in addition to the function name, perhaps add a comment about what mix is testing

e.g
```
Tests
window - relative
minimum_data - absolute
gap - absolute
```

Reply:
Annotated comments on offset tests in [this](https://github.com/FeatureLabs/compose-ml/pull/26/commits/86656326709c219bcd66887016752ea497b42b8a#diff-77d230f17f898fc20eaf609b9d931b91R37) commit.